.travis.yml

```diff
@@ -1,11 +1,11 @@
 # This Travis job script has been generated by a script via
 #
-#   runghc make_travis_yml_2.hs '-o' '.travis.yml' '--ghc-head' '--irc-channel=irc.freenode.org#haskell-lens' '--no-no-tests-no-bench' '--no-unconstrained' '--allow-failure=7.0.4' '--allow-failure=7.2.2' '--config=cabal.haskell-ci' 'cabal.project'
+#   haskell-ci '-o' '.travis.yml' '--config=cabal.haskell-ci' 'cabal.project'
```

Comment:
Is there any way to tell what version of `haskell-ci` generated a particular `.travis.yml` file? I'm thinking of something like what `hpack` does at the top of its generated `.cabal` files:

```
-- This file has been generated from package.yaml by hpack version 0.31.0.
--
-- see: https://github.com/sol/hpack
--
-- hash: cbcda7d66dae228545cd78a42c4c4edf8ec2a746587f3aac59ba8a8775252cc5
```

This sort of information could be useful in case we ever need to debug the generated contents of a `.travis.yml` file.